components/patient-data-sharing/push-api/src/main/resources/ApplicationResources.properties

```diff
@@ -49,19 +45,17 @@ phenotips.PushPatient.male=Male
 phenotips.PushPatient.unknown=Unknown
 phenotips.PushPatient.negative=NO
 phenotips.PushPatient.oneServerMessage=There is only one configured remote server: 
-phenotips.PushPatient.patientUploadedMessage=This patient was uploaded to
-phenotips.PushPatient.patientUploadedLessThanHour=less than an hour ago
-phenotips.PushPatient.patientUploadedDays=days ago
-phenotips.PushPatient.patientUploadedHours=hours ago
-phenotips.PushPatient.patientUploadedAsID=and was assigned patient ID
+phenotips.PushPatient.patientUploadedLessThanHour=This patient was uploaded to __serverID__ less than an hour ago and was assigned patient ID __patientID__
+phenotips.PushPatient.patientUploadedDays=This patient was uploaded to __serverID__ __daysN__ day(s) ago and was assigned patient ID __patientID__
+phenotips.PushPatient.patientUploadedHours=This patient was uploaded to __serverID__ __hoursN__ hour(s) ago and was assigned patient ID __patientID__
 phenotips.PushPatient.patientUploadedUpdate=If data in the record has changed, push to __serverID__ again to update the record there.
 phenotips.PushPatient.updateRemotePatient=Update remote patient
 phenotips.PushPatient.createRemotePatient=Create new remote patient
 phenotips.PushPatient.loadingServers=Loading server list\u2026
 phenotips.PushPatient.serverDesc=Selected server does not allow updating existing patients, so can only push again to a new patient record.
 phenotips.PushPatient.successfulUpdate=Updated patient successfully.
-phenotips.PushPatient.successfulPush=Pushed patient successfully, ID of the new patient on the remote server is
-phenotips.PushPatient.pushFailedUpdatesDisabled=Unable to update this patient - updates are disbaled on the remote server
+phenotips.PushPatient.successfulPush=Pushed patient successfully, ID of the new patient on the remote server is __patientID__
```

Comment:
Thanks @veronikaslc . My suggestions:
```
phenotips.patientSharing.push.dialog.consents.warning=Consents that need to be granted before pushing could not be found. This might prevent you from submitting this patient.
phenotips.PushPatient.serverAndUserFirstMessage=You must be logged in to see these options.
phenotips.PushPatient.pushFailedInvalidGuid=Updating failed because the corresponding patient record to update could not be found on {0}. It is possible that the record was deleted from {0}.
```
I'm not sure about this last one, maybe Felicia should chime in.

I don't think this next one is actually used anywhere anymore, I can't find it:
```
phenotips.PushPatient.oneServerMessage=There is only one configured remote server:
```